src/test/rustdoc/const-generics/const-evaluatable-checked.rs

```diff
@@ -0,0 +1,5 @@
+#![crate_name = "foo"]
+#![feature(const_evaluatable_checked, const_generics)]
+#![allow(incomplete_features)]
+// make sure that `ConstEvaluatable` predicates dont cause rustdoc to ICE #77647
+pub struct Ice<const N: usize> where [(); N + 1]:;
```

Comment:
This doesn't actually do anything, you need to either move it to rustdoc-ui and mark it //check-pass or add a `@has` check. I'd prefer you add the check, I have no idea what this displays as.

Reply:
before I made the changes having this was causing `x.py test src/test/rustdoc/const-generics` to fail so im a bit confused heh, will add a `@has` so that can tell what it renders as